### PR TITLE
Adding endpoint to retrieve list of person

### PR DESCRIPTION
**Pull request by josenavas**
Useful to pre-populate dropdown menus, etc,

### CODE REVIEW

**Comment by josenavas** [comment]
@ElDeveloper @antgonza available for a review?

File: qiita_pet/handlers/rest/study_person.py [hunk 1]
```diff
@@ -15,18 +17,26 @@
...
+            try:
+                p = StudyPerson.from_name_and_affiliation(name, affiliation)
+            except QiitaDBLookupError:
+                self.fail('Person not found', 404)
+                return
+            self.write({'address': p.address, 'phone': p.phone,
+                        'email': p.email, 'id': p.id})
+            self.finish()
+        else:
+            self.fail('Missing parameter', 400)
```
Comment (ElDeveloper) [inline qiita_pet/handlers/rest/study_person.py]: I think this should be a raise `MissingArgumentError`?

Comment (josenavas) [inline qiita_pet/handlers/rest/study_person.py]: Thanks! Forgot about that one :)

File: qiita_pet/test/rest/test_study_person.py [hunk 2]
```diff
@@ -45,6 +54,11 @@ def test_get_invalid_query_string(self):
                             headers=self.headers)
         self.assertEqual(response.code, 400)
 
+    def test_get_invalid_query_string_2(self):
+        response = self.get('/api/v1/person?affiliation=knight%20lab',
+                            headers=self.headers)
+        self.assertEqual(response.code, 400)
```
Comment (ElDeveloper) [inline qiita_pet/test/rest/test_study_person.py]: I think this will need to be updated.

Comment (josenavas) [inline qiita_pet/test/rest/test_study_person.py]: Nope - the [response code of a missing argument is 400](http://www.tornadoweb.org/en/stable/web.html#tornado.web.MissingArgumentError) :-) I was basically replicating the behavior of that exception with less readable code xD

Comment (ElDeveloper) [inline qiita_pet/test/rest/test_study_person.py]: 👍

**Pull request merged**